livre/Eq/Client.cpp

```diff
@@ -91,6 +91,14 @@ bool Client::_parseArguments( const int32_t argc, char **argv )
         return false;
     }
 
+#ifdef LIVRE_USE_RESTCONNECTOR
+    if( !_networkParameters.initializeParameters( argc, argv ))
```

Comment:
1) initialize is fishy
2) if really needed, initialize() is enough, otherwise 2x redundant "parameters"